AccountingSystem/private/DataCache.py

```diff
@@ -43,7 +43,7 @@ def getReportData( self, reportRequest, reportHash, dataFunc ):
     Get report data from cache if exists, else generate it
     """
     reportData = self.__dataCache.get( reportHash )
-    if reportData == False:
+    if reportData is None:
```

Comment:
None is a singleton object in Python. So, "is None" comparison is more explicit.